Library/Homebrew/utils/shell.rb

```diff
@@ -40,14 +40,31 @@ def profile
       SHELL_PROFILE_MAP.fetch(preferred, "~/.bash_profile")
     end
 
+    def prepend_set_variable_in_profile(variable, value)
```

Comment:
It seems like `prepend_path_in_profile(path)` should just be `prepend_set_variable_in_profile(PATH, path)`?

Reply:
it can be (and this PR did that before). The difference between `prepend_set_variable` and `prepend_path` is that when `PATH` is unset users most likely won't be able to run `brew` at all. But this is an assumption, of course, and it is more reliable to use `prepend_set`

Reply:
> But this is an assumption, of course, and it is more reliable to use prepend_set

I think we can assume that all Homebrew's users have a `PATH`.